src/fragments/lib/project-setup/flutter/upgrade-guide/upgrade-guide.mdx

```diff
@@ -0,0 +1,70 @@
+With the release of v1 amplify-flutter now supports Web and Desktop for Auth, API, Analytics, and Storage use cases.  Developers can now build cross-platform Flutter apps with Amplify that target iOS, Android, Web, and Desktop (macOS, Windows, Linux) using a single codebase.
+
+We have re-written our libraries in Dart. In some places, we have made breaking changes to improve ergonomics or enable features that had been missing from the v0 implementations.
+
+## Prerequisites
+
+* A Flutter application targeting Flutter SDK >= 3.0.0 with Amplify libraries integrated
+  
+  The following are also required, depending on which platforms you are targeting:
+
+  * An iOS configuration targeting at least iOS 13.0 and XCode version >=13.2
+  * An Android configuration targeting at least Android API level 24 (Android 7.0) or above
+  * Any browser supported by Flutter for Web (you can check the list of supported browsers [here](https://docs.flutter.dev/development/platform-integration/web/faq#which-web-browsers-are-supported-by-flutter))
+  * Any Windows OS meeting Flutter minimums
+  * macOS version 10.15 or higher
+  * Any Ubuntu Linux distribution meeting Flutter minimums
+  * For a full example please follow the [project setup walkthrough](/lib/project-setup/create-application)
+
+## Auth
+
+- Changes to API options
+- Changes to exception handling
+- Changes to nextStep types
+- getPlugin for Cognito-specific behavior
+- Changes to Hub events
+
+## Analytics
+
+- `AnalyticsProperties` renamed to `CustomProperties` 
+- `AnalyticsUserProfile` renamed to `UserProfile`
+- `AnalyticsUserProfileLocation` renamed to `UserProfileLocation`
+- In v1 you can now specify autoFlushEventsInterval in seconds in amplifyConfiguration.dart to change how often cached events are flushed. v0 does not support this due to differences in iOS and Android implementations.
+- Previously Analytics was built on top of existing Amplify Android and iOS functionality. Those platforms behaved differently but now behave the same:
+    - in v1 flushing occurs at 30 second intervals. In v0 default auto flush iOS was 60 seconds, Android was 30 seconds.
+    - in v1 session pause and resume are not automatically tracked. In v0 iOS auto tracked session pause and resume events. 
+- You can now save to the `UserAttributes` field of a Pinpoint Endpoint by sending a `AWSPinpointUserProfile` to `identifyUser`.
+- Analytics Endpoint metadata is stored in a different namespace/location.  Existing metadata will automatically be migrated.
+- Analytics cached event data is now stored differently. Existing cached analytics events will not be migrated.
+- There are now typed `AnalyticsException`s for specific exception cases. 
```

Comment:
```suggestion
- There are now typed `AnalyticsException`s for specific exception cases. See [the documentation on Pub](https://pub.dev/documentation/amplify_analytics_pinpoint/latest/amplify_analytics_pinpoint/amplify_analytics_pinpoint-library.html) for a full list of exceptions.
```

Lets link to the docs on pub. It clearly shows the full list. That link directs to the latest stable docs, which right now is 0.6.13, but will be 1.0 after we publish.

Note: I am not sure if that link is formatted correctly for the docs site.

Reply:
link format worked locally for me, update coming